src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Application {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
+
+        Figure[] figures = {
+                supplier.getRandomFigure(),
```

Comment:
Instead of repeating 'supplier.getRandomFigure()' multiple times, consider using a loop to generate random figures 🔄